NLogWrapper.cs

```diff
@@ -58,7 +58,7 @@ private void initializeLogger(LoggerType loggerType, ShortenerSettings settings)
 
     private string buildTags()
     {
-        return String.Format("env:{0},service:{1}", shortenerSettings.env, serviceName);
+        return String.Format("env:{0},service:{1},assemblyversion:{2}", shortenerSettings.env, serviceName, shortenerSettings.version);
```

Comment:
This is the Datadog tag right? If so they recommend `version`

Reply:
Makes sense, thanks. My original idea was to split out the versions since TinyBlazorAdmin has two projects, one for the functions and one for the utilities. Later in I figured we didn't really need both versions reporting but didn't simplify the name